cypress/support/fragments/eholdings/eHoldingsPackages.js

```diff
@@ -109,7 +109,7 @@ export default {
   getCustomPackageViaApi: () => {
     cy.okapiRequest({
       path: 'eholdings/packages',
-      searchParams: { 'filter[custom]': true, count: 10, pageSize: 10 },
+      searchParams: { 'filter[custom]': true /* count: 10, pageSize: 10 */ },
```

Comment:
Please remove commented code and leave comment why parameters were removed.

Reply:
removed and added explaining comment